actors/runtime/src/main/java/cloud/orbit/actors/Stage.java

```diff
@@ -711,7 +711,7 @@ public void setBroadcastActorDeactivations(boolean broadcastActorDeactivation)
         }
         if (messageSerializer == null)
         {
-            messageSerializer = new JavaMessageSerializer();
+            messageSerializer = new KryoSerializer();
```

Comment:
Can we use a shared kryo serializer for messaging and object cloning?

Reply:
We don't do this for the loopback cloner. Do you see a big advantage and sharing across all 3?

Reply:
Lower memory footprint.